app/concierge/mycore/templates/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head><link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <meta charset="UTF-8">
+    <title>{{ title }}</title>
+</head>
+<body>
+<!--Navbar-->
+<nav class="bg-dark navbar navbar-expand-lg navbar-dark primary-color">
+
+  <!-- Navbar brand -->
+  <a class="navbar-brand" href="/">HomePage</a>
+
+  <!-- Collapse button -->
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#basicExampleNav"
+    aria-controls="basicExampleNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <!-- Collapsible content -->
+  <div class="collapse navbar-collapse" id="basicExampleNav">
+
+    <!-- Links -->
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/journals">Journals</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/tenants">Tenants</a>
+      </li>
+        <li class="nav-item">
+        <a class="nav-link" href="/rooms">Rooms</a>
+      </li>
+    </ul>
+      </div>
+      </nav>
+    <!-- Links -->
+    {% block content %}
+
+      <div align="center">
+          <h1 >{{ title }}</h1>
+          <p><a href="/journal"><h2>Add Journal</h2></a></p>
```

Comment:
instead of hardcoding url use `url route_name`
`<a href="{% url products:index %}"> Products </a>`

Reply:
Thanks for idea))